tests/scripts/all.sh

```diff
@@ -2435,7 +2283,7 @@ component_test_psa_crypto_config_accel_ecc () {
     # Configure and build the main libraries
     # ---------------------------------------
 
-    # start with default + driver support
+    # start with default (no TLS 1.3 or USE_PSA) + driver support
```

Comment:
(Commenting not on this line but on the component.) Do we still need this component, though? I tend to think we're OK with three components for individual modules (`accel_pake`, `accel_ecdsa` and `accel_ecdh`) not using PSA, and then the component added by this PR accelerating all 3 and `USE_PSA`.

Do you see any gain in having all 3 accelerated without `USE_PSA` in addition?

Reply:
You are right, we don't need this since `component_test_psa_crypto_config_[accel/reference]_all_ec_algs_use_psa()`  has better coverage